apps/judicial-system/web/src/routes/Court/InvestigationCase/Confirmation/Confirmation.strings.ts

```diff
@@ -0,0 +1,40 @@
+import { defineMessages } from 'react-intl'
+
+export const confirmation = defineMessages({
+  title: {
+    id: 'judicial.system.court:confirmation.title',
```

Comment:
Is it not the name space that is making it unique?
```suggestion
    id: 'judicial.system.core:confirmation.title',
```